config/default/manager_auth_proxy_patch.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
   namespace: system
 spec:
   template:
+    metadata:
```

Comment:
I don't think we use this config for generating openshift 4.8 manifest.  /cc @pliurh 
The change is not needed.

Reply:
Correct.